draft-ietf-add-split-horizon-authority.xml

```diff
@@ -280,11 +280,16 @@
 
     <section anchor="establishing">
       <name>Establishing Local DNS Authority</name>
-      <t>To establish its authority over some DNS zone, a participating
-      network <bcp14>MUST</bcp14> offer one or more encrypted resolvers via DNR <xref
-      target="I-D.ietf-add-dnr"/> or an equivalent mechanism (see <xref
-      target="vpn"/>). At least one of these resolvers' Authentication
-      Domain Names (ADNs) <bcp14>MUST</bcp14> appear in an NS record for that zone. This
+      <t>To establish its authority over some DNS zone, a
+      participating network <bcp14>MUST</bcp14> offer one or more
+      encrypted resolvers via DNR <xref target="I-D.ietf-add-dnr"/>,
+      DDR <xref target="I-D.ietf-add-dnr"/>, or an equivalent mechanism
+      (see <xref target="vpn"/>).  If the encrypted resolver is
+      identified by name (e.g., DNR), at least one of these resolvers'
+      Authentication Domain Names (ADNs) <bcp14>MUST</bcp14> appear in
+      an NS record for that zone.  If the encrypted resolver is
+      identified by IP address (e.g., DDR), at least one of its TLS
+      Subject Alternative Names MUST match one of the ADNs.  This
```

Comment:
In case of DDR, ADN is not present. You may want to update the above line related to DDR as follows;
If the encrypted resolver is identified by IP address (e.g., DDR), at least one of the subjectAltName entries in the resolver certificate MUST appear in an NS record for that zone. 

I think we should also say the client must first perform Verified Discovery (discussed in https://www.ietf.org/archive/id/draft-ietf-add-ddr-08.html#section-4.2) before performing the above check. In other words, opportunistic discovery mode cannot be used by the client (see https://www.ietf.org/archive/id/draft-ietf-add-ddr-08.html#section-4.3).

Reply:
Right, having a name from the cert of the DDR server seems like a reasonable thing to tie to the NS record. It's not what the discovery was based on (which is an IP address), but it tells you that the resolver you can use is related to the zone.